src/Config/ObjectModel/EntityRestOptions.cs

```diff
@@ -19,4 +19,5 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 public record EntityRestOptions(SupportedHttpVerb[] Methods, string? Path = null, bool Enabled = true)
 {
     public static readonly SupportedHttpVerb[] DEFAULT_SUPPORTED_VERBS = new[] { SupportedHttpVerb.Get, SupportedHttpVerb.Post, SupportedHttpVerb.Put, SupportedHttpVerb.Patch, SupportedHttpVerb.Delete };
+    public static readonly SupportedHttpVerb[] DEFAULT_SUPPORTED_VERBS_FOR_SP = new[] { SupportedHttpVerb.Post };
```

Comment:
nit: do we want to have this named "supported"?

Reply:
updated